exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
+let myname = "Timea    ";
```

Comment:
You need to use Camelcase for the name of the variable.

Reply:
Thanks, I did't notice. Fixed it.